openproblems/tasks/chromatin_potential/checks.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def check_dataset(adata):
+    assert "mode2" in adata.obsm
+    assert "mode2_obs" in adata.uns
+    assert "mode2_var" in adata.uns
+    assert "mode2_var_chr" in adata.uns
+    assert "mode2_var_start" in adata.uns
+    assert "mode2_var_end" in adata.uns
```

Comment:
Missing the following:

* `adata.uns['species']`
* `adata.uns['release']`
* `adata.var['chr']`
* `adata.var['start']`
* `adata.var['end']`

Reply:
I will fill the missing attributes.